articles/tokens/access_token.md

```diff
@@ -1,49 +1,115 @@
+
+There is a [sample for altering scopes in a Rule](https://github.com/auth0/rules/blob/dff2a3e72f01d33af3086414be7cf115b19eea0c/rules/custom-scopes.md)
+
+Additional information on the `access_token` is [here](/jwt)
+
+A valid JWT can be pasted into the [jwt.io website](https://jwt.io) to view the contents of the JWT.
+
+A [blog entry on JWT](https://auth0.com/blog/2015/07/21/jwt-json-webtoken-logo/) provides an explanation of why it is getting to be more popular.
+
+[Additional samples](https://github.com/auth0/auth0.js) show use of the auth0.js library.
 
 ## Validity
 
-The Auth0 access token remains valid for 24 hours.
+In the case of the [authorization code grant](/api-auth/grant/authorization-code) and the [implicit code grant](/api-auth/grant/implicit) flows the `access_token` is valid for ???
+
+In the case of the [client credentials flow](/api-auth/grant/client-credentials), the `access_token` is valid for 8000 seconds (roughly 2.25 hours) by default.  !!The expiration of this token can be set in the `Apps/APIs` -> Settings screen using the `JWT expiration` field!!
+
+The validity period of the token can be configured per [Resource Server](??)
 
 ## Renewing the token
 
-There is currently no way to renew the Auth0 access token without triggering authentication by the user.  To get a new access token, simply follow the process for getting an access token as described above in the How to Get an Access Token section.
+?? How do we get a refresh token ??
+?? How do we use the refresh token to obtain a new access token ??
 
 ## Termination of the token
 
-Once issued, Auth0 access tokens can not be revoked.  
+Once issued, tokens can not be revoked in the same fashion as cookies with session id’s for server-side sessions.  As a result, tokens should be issued for relatively short periods, and then renewed periodically if the user remains active.  See the above section on renewing access tokens.
 
 ## Uses
 
-The Auth0 access token is used to call the Auth0 Management APIv1.  This version of the Auth0 Management API has been deprecated and should no longer be used.
+The `access_token` is designed to be used to pass information about a user between websites,  web programs and APIs in an industry standard, URL-friendly fashion.  One advantage of using an `access_token` for this purpose is that the recipient can validate the token without having to make a call back to the issuer of the token.  The token is also designed to enable being passed from one web property to another, via an untrusted client, such that the client cannot alter the token without such tampering being evident to the recipient.
+
+It can also be used to call the /userinfo endpoint within the authentication API to get user profile information as documented [here](/auth-api#user-profile) (?? is this still valid ??)
```

Comment:
My understanding is that this will still work. The new `access_token` will be accepted by the existing `/userinfo` endpoint and return the full user profile. @jaredhanson is that correct?